src/main/java/it/gov/pagopa/common/web/exception/ServiceException.java

```diff
@@ -0,0 +1,24 @@
+package it.gov.pagopa.common.web.exception;
+
+
+import lombok.Getter;
+
+@Getter
+public class ServiceException extends RuntimeException {
+    private final String code;
+    private final boolean printStackTrace;
+    private final transient ServiceExceptionResponse response;
+
+
+    public ServiceException(String code, String message, ServiceExceptionResponse response) {
+        this(code, message, false, null, response);
+    }
+
+    public ServiceException(String code, String message, boolean printStackTrace, Throwable ex, ServiceExceptionResponse response) {
```

Comment:
If it's not too much expensive, I would prefer to let the Throwable as last parameter, and the response as next to message parameter
In order to have a coherence in the parameter order
```suggestion
    public ServiceException(String code, String message, ServiceExceptionResponse response, boolean printStackTrace, Throwable ex) {
```